app/models/contact.rb

```diff
@@ -2,36 +2,51 @@
 #
 # Table name: contacts
 #
-#  id         :bigint           not null, primary key
-#  email      :string
-#  name       :string
-#  phone      :string
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
-#  team_id    :bigint           default(2), not null
+#  id                  :bigint           not null, primary key
+#  email               :string
+#  name                :string
+#  notes               :text
+#  notes_updated_at    :datetime
+#  phone               :string
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#  notes_updated_by_id :bigint
+#  team_id             :bigint           default(1), not null
 #
 # Indexes
 #
-#  index_contacts_on_phone              (phone)
-#  index_contacts_on_team_id            (team_id)
-#  index_contacts_on_team_id_and_phone  (team_id,phone) UNIQUE
+#  index_contacts_on_notes_updated_by_id  (notes_updated_by_id)
+#  index_contacts_on_phone                (phone)
+#  index_contacts_on_team_id              (team_id)
+#  index_contacts_on_team_id_and_phone    (team_id,phone) UNIQUE
 #
 # Foreign Keys
 #
+#  fk_rails_...  (notes_updated_by_id => users.id)
 #  fk_rails_...  (team_id => teams.id)
 #
 class Contact < ApplicationRecord
   include Conversationalist
   has_one :conversation, dependent: :destroy, touch: true
   has_many :messages, as: :sender, dependent: nil # let the Conversation model delete the messages
   belongs_to :team
+  belongs_to :notes_updated_by, class_name: "User", optional: true
```

Comment:
on pourrait utiliser un autre terme (tout en gardant le même nom de colonne) parce que `contact.notes_updated_by.name` c'est pas évident. Plutôt `contact.notes_last_editor` ou quelque chose comme ça : tu veux pointer vers un user et il faut que grammaticalement ça soit équivalent à user

Reply:
Ca fait bien plus sens effectivement `notes_last_editor_id` que `not notes_updated_by_id`